pandas/tests/groupby/test_groupby.py

```diff
@@ -2134,3 +2134,24 @@ def test_groupby_column_index_name_lost_fill_funcs(func):
     result = getattr(df_grouped, func)().columns
     expected = pd.Index(["a", "b"], name="idx")
     tm.assert_index_equal(result, expected)
+
+
+@pytest.mark.parametrize("func", ["max", "min"])
+def test_groupby_rolling_index_changed(func):
+    ds = pd.Series(
```

Comment:
Could you add `# GH 36018` as a comment?

Reply:
Thank for the review. Forgot the comment. Is added now.